src/corona_hakab_model/analyzers/example_script.py

```diff
@@ -31,3 +32,7 @@
 # you can also plot all the parameters time propagation
 plot_parameter_propagation_aggregated(mean_df=results_df_dict["mean"], std_df=results_df_dict["std"])
 
+
+# plot social circles sickness analisys
+initial_sick_data_df, final_sick_data_df, all_circles_df = load_population_data_to_dfs()
```

Comment:
If you do not use the initial_sick_data_df why not use python temporary variable _ instead?

Reply:
this is an example for usage
I added, another plot for initial_sick_data